File: fixtures/dart/post_escaped_single_quotes_in_double_quotes.dart
import 'package:http/http.dart' as http;

void main() async {
  var data = 'foo=\'bar\'';

  var url = Uri.parse('http://example.com/');
  var res = await http.post(url, body: data);
  if (res.statusCode != 200) throw Exception('http.post error: statusCode= ${res.statusCode}');
  print(res.body);
}

File: fixtures/r/post_escaped_single_quotes_in_double_quotes.r
require(httr)

headers = c(
  `Content-Type` = 'application/x-www-form-urlencoded'
)

data = list(
  `foo` = '\\\'bar\\\''
)

res <- httr::POST(url = 'http://example.com/', httr::add_headers(.headers=headers), body = data)
